frontend/.eslintrc.js

```diff
@@ -7,6 +7,8 @@ const noUnusedVarsOptions = {
     caughtErrors: "all",
     caughtErrorsIgnorePattern: "^_",
 };
+const indentation = 4;
+const indentOptions = [indentation, { SwitchCase: 1 }];
```

Comment:
Why exactly does this need to be its own variable instead of just writing `[4, { SwitchCase: 1 }]`?

Reply:
Because we use it twice, which we don't actually need to, as you rightfully noticed. :wink: